accumulator_updater/tests/accumulator_updater.ts

```diff
@@ -322,16 +304,32 @@ function parseAccumulatorInput({
   data,
 }: {
   header: AccumulatorInputHeader;
-  data: Buffer;
-}): AccumulatorPriceMessage {
-  console.log(`header: ${JSON.stringify(header)}`);
-  if (header.accountSchema === 0) {
-    console.log(`[full]data: ${data.toString("hex")}`);
-    return parseFullPriceMessage(data);
-  } else {
-    console.log(`[compact]data: ${data.toString("hex")}`);
-    return parseCompactPriceMessage(data);
+  data: Uint8Array;
+}): AccumulatorPriceMessage[] {
+  const messages = [];
+
+  let dataBuffer = Buffer.from(data);
+
+  for (let i = 0; i < header.indexes.length; i++) {
+    const inputIndex = header.indexes[i];
+    if (inputIndex.len == 0) {
+      continue;
+    }
+
+    const inputData = dataBuffer.subarray(
+      inputIndex.offset,
+      inputIndex.offset + inputIndex.len
+    );
+    if (i == 0) {
+      messages.push(parseFullPriceMessage(inputData));
```

Comment:
is this temporary (presumably the messages should themselves encode what type of data they are)? maybe worth a comment

Reply:
i've updated this thanks!